### PR TITLE
DM-46122: Replace verify metrics for Completeness and Count based on magnitudes

### PR DESCRIPTION
**Pull request by BrunoSanchez**

### CODE REVIEW

File: pipelines/apDetectorVisitFakesCore.yaml [hunk 1]
```diff
@@ -86,4 +86,3 @@ tasks:
       python: |
         from lsst.analysis.tools.atools import FractionFoundFakesDiaSnrMetric
         from lsst.analysis.tools.atools import FractionFoundFakesDiaMagMetric
-
```
Comment (kfindeisen) [inline pipelines/apDetectorVisitFakesCore.yaml]: Please squash this sort of "fixup" commit into "Added fakes metrics"; [commits should be self-contained units](https://developer.lsst.io/work/flow.html#appendix-commit-organization-best-practices).

File: pipelines/apDetectorVisitFakesCore.yaml
Comment (kfindeisen) [inline pipelines/apDetectorVisitFakesCore.yaml:1]: Please write all commit messages [in the imperative tense](https://developer.lsst.io/work/flow.html#appendix-commit-message-best-practices); this goes for all three PRs.

**Review by kfindeisen — APPROVED** [review]
The code changes themselves look good, modulo some questions that are probably my ignorance of `analysis_tools`. However, please clean up the commit history before merging.

File: pipelines/apDetectorVisitFakesCore.yaml [hunk 2]
```diff
@@ -32,9 +32,20 @@ tasks:
...
+      atools.numFoundFakesDiaMag22t24Metric: FractionFoundFakesDiaMagMetric
+      atools.numFoundFakesDiaMag22t24Metric.magMin: 22
+      atools.numFoundFakesDiaMag22t24Metric.magMax: 24
+      atools.numFoundFakesDiaMag24t26Metric: FractionFoundFakesDiaMagMetric
+      atools.numFoundFakesDiaMag24t26Metric.magMin: 24
+      atools.numFoundFakesDiaMag24t26Metric.magMax: 26
       connections.outputName: diaFakesDetectorVisitCore
       python: |
         from lsst.analysis.tools.atools import FractionFoundFakesDiaSnrMetric
+        from lsst.analysis.tools.atools import FractionFoundFakesDiaMagMetric
```
[the remark this answers is not in the document]
Comment (BrunoSanchez) [reply pipelines/apDetectorVisitFakesCore.yaml:48]: Surprisingly yes. I think this pattern is present in more than one of the metric pipelines in `analysis_tools` and it is a fair question why indeed this is needed.

**Pull request merged**